### DISCUSSION TITLE
Multiplex M-Link not working

### DISCUSSION THREAD

**Discussion by racsnet** [iNavFlight/inav#7396] [General]
Hello there, atm i try to connect my Multiplex RX-9-DR pro to INAV running on MAMBA F405 MK2.
Sadly it does not work. Firmware on my Receiver is 1.38 (latest). I configured my Reciver to SRXLv1 abd SRXLv2. In INAV i tryed XB-B, XB-B-RJ01 and SRXL. But nothing worked. In Cleanflight and Betaflight it works with XBUS-B.

Port used: RX3 (RX1 is used for GPS)

Any idea on this? 
Thanks

**Comment by YannickF** [comment]
HEllo.....
Same problem for me !
It seems that iNav doesn't manage XBUS (XBUS_MOD_B) anymore.

Multiplex SRXL is actually an XBUS protocol.
I'll try to insert the old source code for this protocol in the latest version (8.0.0) but i'm really not sure I'm able to do that to build a specific version for my usage.
